.travis.yml

```diff
@@ -52,8 +57,11 @@ matrix:
       env: ACTIVERECORD=5.0.0
     - rvm: rbx
       env: ACTIVERECORD=5.1.1
+    - rvm: rbx
+      env: ACTIVERECORD=5.2.0
   allow_failures:
     - rvm: rbx
+    - env: ACTIVERECORD=5.2.0
```

Comment:
Doesn't this add all ActiveRecord 5.2.x builds to the failure accept list? If so, this is not what we want.

Reply:
It adds all builds using ACTIVERECORD=5.2.0 env to the failure accept list.
See https://travis-ci.org/attr-encrypted/attr_encrypted/builds/367612719
We need to fix some codes.

> all ActiveRecord 5.2.x

Do you intend this _all_ as 5.2.1 or upper versions? This change does nothing for those versions.

Reply:
Can you please explain how this is useful? I believe it misleads users into thinking that ActiveRecord 5.2 is supported when it is not as the build is clearly failing.

Reply:
I think this is the first step to support rails 5.2.
We need to know the build is failing with rails 5.2 at first.

After fixing some code for supporting rails 5.2, we drop the allow failure line.
What do you think about this workflow.

Reply:
Hmm, I see. Let's do this, we can add ActiveRecord 5.2 to the build matrix, but let's leave it out of the failure accept list. Sure, it'll make the gem look like it's broken, but then people can simply click the build to see that everything but 5.2 is passing. Whereas, if we add it to the failure accept list, they might assume that the build is passing and try to use it with 5.2 not realizing that it's not supported.